assignment-2/index.js

```diff
@@ -1,26 +1,62 @@
 const fs = require('fs');
 const path = require('path');
-
 const { stat } = fs.promises;
+const program = require('commander');
+const tree = require('../assignment-1/index');
+
+program
+  .version('0.0.1')
+  .description('Prints a tree of the directory structure')
+  .option('-d, --depth <n>', 'Depth of the directory tree', 1)
+  .option('value', 'Path to the directory', process.cwd());
+
+program.parse(process.argv);
+
+const { depth, value } = program.opts();
+console.log(depth, 'depth');
+console.log(value, 'path');
+
+function getFiles(dir) {
+  return new Promise((resolve, reject) => {
```

Comment:
you could use fs/promises as well

Reply:
@korzio I was try to create a similar function like promisify. So I tried it with this approach.